frontend/sass/norent/styles.scss

```diff
@@ -61,4 +76,143 @@ $jf-navbar-content: $black;
       color: $link;
     }
   }
+
+  // Default spacing between paragraphs
+  p:not(:last-child) {
+    margin-bottom: 0.5rem;
+  }
+
+  // Set all Bulma container classes to have max width:
+  .container {
+    max-width: 800px;
+    &.jf-tight-container {
+      max-width: 600px;
+    }
+    &.jf-wide-container {
+      max-width: 1000px;
+    }
+  }
+
+  // Make all media icons have vertical centering
+  // as well as smaller margins on mobile
+  .media .media-left,
+  .media .media-right {
+    margin-top: auto;
+    margin-bottom: auto;
+    @media screen and (max-width: $tablet) {
+      figure {
+        margin-left: 1rem;
+        margin-right: 1rem;
+      }
+    }
+  }
+}
+
+// CUSTOM CLASSES
+
+@mixin is-horizontal-center {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.jf-space-below {
+  margin-bottom: 2rem;
+}
+
+.jf-has-text-centered-desktop {
+  @media screen and (min-width: $tablet) {
```

Comment:
Huh, should this actually be called `jf-has-text-centered-tablet`?  I get so confused by these breakpoints.  Also it looks like the latest version of bulma actually has [SASS mixins for this](https://github.com/jgthms/bulma/blob/master/sass/utilities/mixins.sass#L81,L129) but I'm not sure if our out-of-date version has them.

Reply:
ahhh I think you're right... Bulma's breakpoints are uber confusing :/